umbral-pre/src/params.rs

```diff
@@ -13,15 +13,10 @@ pub struct Parameters {
 impl Parameters {
     /// Creates a new parameter object.
     pub fn new() -> Self {
-        let g = CurvePoint::generator();
-        let g_bytes = g.to_array();
-
-        let parameters_seed = b"NuCypher/UmbralParameters/u";
-
         // Only fails with a minuscule probability,
         // or if the size of a point is too large for the hasher.
         // In any case, we will notice it in tests.
-        let u = unsafe_hash_to_point(&g_bytes, parameters_seed).unwrap();
+        let u = unsafe_hash_to_point(b"NuCypher/UmbralParameters/u").unwrap();
```

Comment:
Why are you removing the input `g` from here? The high-level idea is that `u = H(g, "NuCypher/UmbralParameters/u")`; the motivation is that there's always a unique `u` for any choice of `g`.

Reply:
But we didn't have a choice of `g`. It's always the generator. Or do you want to plan for an eventuality where we have several sets of parameters? In that case, why specifically use the generator and not just another bytestring?

Reply:
The fact that there's not a choice of `g` is purely an implementation decision. The umbral scheme can potentially use any curve point as `g`. By having `u` depend on `g` in a non-cryptographically sensitive manner gives you freedom to choose `g` and not worry about `u`.

Reply:
Ok, rolled back `g` usage and added a TODO for #35 in case we ever want to use the standard hashing method.